presto-cli/src/main/java/com/facebook/presto/cli/StatusPrinter.java

```diff
@@ -382,8 +392,19 @@ private void printStageTree(StageStats stage, String indent, AtomicInteger stage
             rowsPerSecond = formatCountRate(0, new Duration(0, SECONDS), false);
         }
         else {
-            bytesPerSecond = formatDataRate(bytes(stage.getProcessedBytes()), elapsedTime, false);
-            rowsPerSecond = formatCountRate(stage.getProcessedRows(), elapsedTime, false);
+            ProgressStatus previousStage;
+            if (previousStageStatuses.containsKey(id)) {
+                previousStage = previousStageStatuses.get(id);
```

Comment:
Consider use `previousStageStatus` or `previousStatus`? It's a Status, not a Stage :)